tools/javadoc-style/src/main/java/io/spine/tools/javadoc/style/LineFormatting.java

```diff
@@ -49,15 +49,18 @@
      */
     @Override
     public String execute(String text) {
-        List<String> textAsLines = Splitter.on(lineSeparator())
-                                           .splitToList(text);
-        List<String> formattedLines = newLinkedList();
-        for (String line : textAsLines) {
-            String formattedLine = formatLine(line);
-            formattedLines.add(formattedLine);
-        }
-        return Joiner.on(lineSeparator())
-                     .join(formattedLines);
+        String separator = lineSeparator();
+        List<String> lines =
+                Splitter.on(separator)
+                        .splitToList(text);
+        List<String> formattedLines =
+                lines.stream()
+                     .map(this::formatLine)
+                     .collect(toList());
+        String result =
+                Joiner.on(separator)
+                      .join(formattedLines);
+        return result;
```

Comment:
`Splitter` and `Joiner` instances are thread-safe and it is usually recommended to cache them as there's no need to create them each time the method is called. Especially taking into account the static nature of the `separator` here.
Also, `Splitter` has a convenient `splitToStream` method which we can use instead of creating a list first and then converting it to a stream and back to the list.

```
        String separator = System.lineSeparator(); 
        String result = Splitter.on(separator)
                .splitToStream(text)
                .map(this::formatLine)
                .collect(Collectors.joining(separator));
```